cfpb/cfgov-refresh
# Upgrade to latest cf-core to get Demibold web font cross-browser fixes

Fixes https://GHE/CFGOV/platform/issues/2484

See https://github.com/cfpb/capital-framework/pull/778 for extreme amounts of explanation and browser testing screenshots.



## Changes

- bump version of `cf-core`

## Testing

1. check this out
1. set line 30 of `main.less` to `@use-font-cdn: true;` to get Avenir web fonts loaded locally
1. confirm that h5 and h6 elements on pages like http://localhost:8000/ask-cfpb/what-should-i-do-when-a-debt-collector-contacts-me-en-1695/ no longer look exceptionally heavy, especially in Chrome and Firefox.

## Screenshots

### Before, Firefox, Mac
![screen shot 2018-03-14 at 5 07 05 pm](https://user-images.githubusercontent.com/702526/37431140-c6225060-27aa-11e8-8025-4fbba500ec59.png)

### After, Firefox, Mac
![screen shot 2018-03-14 at 5 06 53 pm](https://user-images.githubusercontent.com/702526/37431141-c636cb3a-27aa-11e8-8903-26f7e3597e1a.png)

many more screenshots at https://github.com/cfpb/capital-framework/pull/778

## Notes

- this change is @nataliafitzgerald approved!!!

## Todos

-

## Checklist
